src/pages/studyView/StudyViewPage.tsx

```diff
@@ -75,62 +76,58 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
-        return (
-            <div className="studyView">
-                <div className="topBanner">
-                    {
-                        this.store.studyMetaData.isComplete && (
-                            <div className="studyViewHeader">
-                                <h3>{cancerStudy.name}</h3>
-                                <p dangerouslySetInnerHTML={{__html: cancerStudy.description}}></p>
-                            </div>
-                        )
-                    }
-                </div>
-                <MSKTabs id="studyViewTabs" activeTabId={this.props.routing.location.query.tab}
-                         className="mainTabs">
-
-                    <MSKTab key={0} id="summaryTab" linkText="Summary">
-                            {
-                                this.store.defaultVisibleAttributes.isComplete && 
-                                this.store.initalClinicalDataCounts.isComplete &&
-                                this.store.initialized && 
-                                (
-                                    <div className={styles.studyViewFlexContainer}>
-                                        {this.store.visibleAttributes.map(this.renderAttributeChart)}
-                                    </div>
-                                )
-                            }
-                            {
-                                <div className={styles.studyViewFlexContainer}>
-                                    {this.store.survivalPlotData.result.map(this.renderSurvivalPlot)}
+        if (this.store.studies.isComplete) {
+            return (
+                <div className="studyView">
+                    <div className="topBanner">
+                        <div className="studyViewHeader">
+                            <If condition={this.store.studies.result.length === 1}>
+                                <div>
+                                    <h3>{this.store.studies.result![0].name}</h3>
+                                    <p dangerouslySetInnerHTML={{ __html: this.store.studies.result![0].description }}></p>
                                 </div>
-                            }
+                            </If>
+                            {/*TDOD: currently show as Multiple Studies but should be shandles properly, i.e as in production*/}
+                            <If condition={this.store.studies.result!.length > 1}>
+                                <h3>Multiple Studies</h3>
+                            </If>
+                        </div>
+                    </div>
+                    <MSKTabs id="studyViewTabs" activeTabId={this.props.routing.location.query.tab}
+                        className="mainTabs">
+
+                        <MSKTab key={0} id="summaryTab" linkText="Summary">
+                            <div className={styles.studyViewFlexContainer}>
+                                {this.store.initialClinicalDataCounts.isComplete && 
```

Comment:
so, initialClinicalDataCounts will ensure the presenc eof visisbleAttributes, yes?  because onResult populates it?

Reply:
yes